src/app/api/getStakerPoints.ts

```diff
@@ -0,0 +1,29 @@
+import { encode } from "url-safe-base64";
+
+import { apiWrapper } from "./apiWrapper";
+
+export interface StakerPoint {
+  staker_btc_pk: string;
+  polkadots: number;
+}
+
+export const getStakerPoints = async (
```

Comment:
```suggestion
export const getStakersPoints = async (
```

As we can either pass a single one or an array.